packages/@aws-cdk/pipelines/test/codepipeline/codebuild-step.test.ts

```diff
@@ -125,8 +125,8 @@ test('timeout from defaults can be overridden', () => {
 test('envFromOutputs works even with very long stage and stack names', () => {
   const pipeline = new ModernTestGitHubNpmPipeline(pipelineStack, 'Cdk');
 
-  const myApp = new AppWithOutput(app, 'Alpha'.repeat(20), {
-    stackId: 'Stack'.repeat(20),
+  const myApp = new AppWithOutput(app, 'Alpha'.repeat(10), {
+    stackId: 'Stack'.repeat(10),
```

Comment:
Why change this?

Reply:
This test was failing because it will create a stack with a 200 character name :p